packages/build-tools/src/steps/functions/installPods.ts

```diff
@@ -0,0 +1,18 @@
+import { BuildFunction } from '@expo/steps';
+import spawn from '@expo/turtle-spawn';
+
+export function createInstallPodsBuildFunction(): BuildFunction {
+  return new BuildFunction({
+    namespace: 'eas',
+    id: 'install_pods',
+    name: 'Install Pods',
+    fn: async (stepsCtx, { env }) => {
+      stepsCtx.logger.info('Installing pods');
+      await spawn('pod', ['install'], {
```

Comment:
Should we use the `npx pod-install` thing? https://linear.app/expo/issue/ENG-10529/discuss-replace-pod-install-step-with-npx-pod-installlatest-on-eas

Reply:
I think we should keep it 1:1 with EAS Build standard flow which is currently using `pod install`